lib/spack/spack/config.py

```diff
@@ -119,13 +119,27 @@
 #: Type used for raw YAML configuration
 YamlConfigDict = Dict[str, Any]
 
+#: prefix for name of included configuration scopes
+INCLUDE_SCOPE_PREFIX = "include"
+
+#: safeguard for recursive includes -- raise an error after this many layers.
+MAX_RECURSIVE_INCLUDES = 100
```

Comment:
reduce?

Reply:
@haampie Not sure what you're asking to be changed here wrt limiting the depth of recursion (or includes). The comment? The variable?